build/pipelines/templates-v2/steps-inject-secrets.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  - name: githubClientSecret
+    type: string
+    default: 'FineKeepYourSecrets'
+
+steps:
+  - pwsh: |-
+      $header = Get-Item src/cascadia/QueryExtension/WindowsTerminalIDAndSecret.h -ErrorAction:Ignore

```

Comment:
we check that it exists so that I can merge this PR without breaking unless the file exists ;P

Reply:
You could also call `Get-Content` immediately and check if the result is truthy.

Reply:
This is true.